android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -195,26 +200,26 @@ public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceReque
 
       @Override
       public void onPageFinished(WebView view, final String location) {
-        if (appUrlConfig != null) {
+        if (appUrlConfig != null || appLoadHostsInWebViewConfig != null) {
```

Comment:
This will inject the Capacitor js in the external loaded url, is that desired?

Reply:
@jcesarmobile exactly, my use case perfectly aligns with that – so I don't have to bundle `Capacitor` in the website's `package.json` and has `Capacitor` object ready to use right away!